app/person.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'nameable'
+
```

Comment:
### OPTIONAL 

- [ ]  To run your app, I added the require_relative script to import the capitalize_decorator and trimmer_decorator,  after doing this the app works successfully, you can implement this so your app runs properly 👍🏿

![Screenshot_134](https://github.com/martinkarugaba/ruby-oop-school-library/assets/69638013/b229b41b-16ef-42ab-bad2-e8f0362efce7)

Reply:
Thanks @prolajumokeoni. I'll implement that

Reply:
Hi @prolajumokeoni, my project has not been marked as submit on the dashboard, is there an issue?